src/modules/display.js

```diff
@@ -0,0 +1,44 @@
+import { getData, postData } from './add';
+
+function Score(score, name) {
```

Comment:
- [ ] As per requirements, kindly consider using arrow functions,  instead of the function keyword.
- [OPTIONAL] Also consider using `score`  as we prefer lowercase for singular names for variables and functions as per convention.